src/ORM/Search/SearchContext.php

```diff
@@ -172,6 +172,10 @@ private function search(DataList $query): DataList
      */
     private function prepareQuery($sort, $limit, ?DataList $existingQuery): DataList
     {
+        // @todo In CMS 6 change method param from `$limit = false` to `array|string|null $limit = null`
```

Comment:
Sorry - just realised this should be in `getQuery()` - not in `prepareQuery()`

Reply:
Updated